src/serial/remote/mod.rs

```diff
@@ -25,6 +25,22 @@ mod protocol_v3;
 mod protocol_v4;
 pub mod riscv_debug;
 
+pub struct RemoteCommands;
+
+#[allow(unused)]
+impl RemoteCommands
+{
+	/// This command asks the probe if the power is used
```

Comment:
nit: "if power is being supplied to the target"

Reply:
Agreed, and changed.